internal/olm/operator/bundle/install.go

```diff
@@ -116,3 +118,13 @@ func loadBundle(ctx context.Context, bundleImage string) (registryutil.Labels, *
 
 	return labels, bundle.CSV, nil
 }
+
+func getSupportedInstallModes(csvInstallModes []v1alpha1.InstallMode) sets.String {
+	supported := sets.NewString()
+	for _, im := range csvInstallModes {
+		if im.Supported {
+			supported.Insert(string(im.Type))
+		}
+	}
+	return supported
+}
```

Comment:
We should use this logic for `run packagemanifests` as well right?

Can we make the operator group handling fully internal to the `OperatorInstaller` struct?

Reply:
@joelanford I started to go down that path but I needed to pass the CSV down to the OperatorInstaller. So I opted to use the csv we have when loading the bundle to populate the supported install modes. Basically only passing down what I need vs the whole CSV.